onnxsim/onnx_simplifier.py

```diff
@@ -130,10 +131,13 @@ def is_dynamic(node):
     return copy.deepcopy(const_nodes)
 
 
-def forward(model, inputs=None, input_shapes: Optional[TensorShapes] = None) -> Dict[str, np.ndarray]:
+def forward(model, inputs: Optional[dict] = None, input_shapes: Optional[TensorShapes] = None,
+            custom_lib: Optional[str] = None) -> Dict[str, np.ndarray]:
     if input_shapes is None:
         input_shapes = {}
     sess_options = rt.SessionOptions()
+    if os.path.exists(custom_lib):
+        sess_options.register_custom_ops_library(custom_lib)
```

Comment:
It will be better to raise an exception when the file doesn't exist

Reply:
great, i'll add an exception in there